src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml

```diff
@@ -220,12 +220,13 @@
                     <Setter Property="AutomationProperties.Name" Value="{Binding AutomationName}"/>
                     <Setter Property="IsEnabled" Value="False"/>
                     <Setter Property="Background" Value="{DynamicResource {x:Static vsui:EnvironmentColors.TitleBarInactiveBrushKey}}"/>
+                    <Setter Property="MinHeight" Value="22"/>
```

Comment:
Found this value is missed (we have three MenuItem template here, two of them has MinHeight = 22)
And it is causing all the height of Header is 18, and the height of other menuItem is 22.
After:
![image](https://user-images.githubusercontent.com/24360909/128989443-5a121a23-da8c-4e08-b055-f934a47f8b80.png)

Before:
![image](https://user-images.githubusercontent.com/24360909/128989833-64de98b6-009d-4df4-b322-e9bd849a5761.png)

Note: there is a small edge between the icon and the text before. Make all the min height to 22 will resolve the problem